src/main/java/everyos.browser.webicitybrowser/everyos/browser/webicitybrowser/gui/ui/OverlyingBlockComponentUI.java

```diff
@@ -1,72 +1,103 @@
 package everyos.browser.webicitybrowser.gui.ui;
 
+import everyos.browser.javadom.imp.TimeUtil;
 import everyos.browser.webicitybrowser.gui.Styling;
+import everyos.browser.webicitybrowser.gui.component.CircularText;
 import everyos.browser.webicitybrowser.gui.component.OverlyingBlockComponent;
+import everyos.browser.webicitybrowser.util.TimeSystem;
 import everyos.engine.ribbon.core.component.Component;
+import everyos.engine.ribbon.core.directive.PositionDirective;
+import everyos.engine.ribbon.core.directive.SizeDirective;
+import everyos.engine.ribbon.core.event.UIEvent;
 import everyos.engine.ribbon.core.rendering.Renderer;
+import everyos.engine.ribbon.core.shape.Dimension;
+import everyos.engine.ribbon.core.shape.Location;
 import everyos.engine.ribbon.core.shape.Rectangle;
+import everyos.engine.ribbon.core.shape.SizePosGroup;
 import everyos.engine.ribbon.core.ui.ComponentUI;
+import everyos.engine.ribbon.core.ui.UIDirective;
 import everyos.engine.ribbon.core.ui.UIManager;
 import everyos.engine.ribbon.ui.simple.SimpleBlockComponentUI;
+import everyos.engine.ribbon.ui.simple.appearence.Appearence;
+import everyos.engine.ribbon.ui.simple.helper.ComputedChildrenHelper;
+import everyos.engine.ribbon.ui.simple.helper.StringWrapHelper;
+import everyos.engine.ribbon.ui.simple.layout.InlineBlockLayout;
+import everyos.engine.ribbon.ui.simple.layout.InlineLayout;
 
 public class OverlyingBlockComponentUI extends SimpleBlockComponentUI {
 
+    private Appearence appearence;
     private OverlyingBlockComponent component;
-    private Component contentPane;
-    private ComponentUI contentPaneUI;
+    private InlineBlockLayout layout;
 
     public OverlyingBlockComponentUI(Component c, ComponentUI parent) {
         super(c, parent);
-    }
-
-    //TODO: Fix merge conflicts
-    
-    /*@Override
-    protected ComponentUI[] calcChildren(UIManager uimgr) {
-        component = getComponent();
-        if (component.getChildren().length < 1) return new ComponentUI[0];
-        contentPane = component.getChildren()[0];
-        contentPane.unbindAll();
-        contentPaneUI = uimgr.get(contentPane, getParent());
-        contentPane.bind(contentPaneUI);
-        return new ComponentUI[]{contentPaneUI};
+        component = (OverlyingBlockComponent) c;
+        this.appearence = new OverlyingBlockComponentUIAppearence();
+        layout = (InlineBlockLayout) getLayout();
+        layout.setConsiderChildren(false);
```

Comment:
Can we rename this to setAutoManageChildren?

Reply:
Yeah we can.
In essence, when we say "consider" it also means that we will not define layouts out calculated them for children. But here we are not really doing anything with their layouts so... Okay 👌